index.js

```diff
@@ -67,7 +73,6 @@ module.exports = {
     'property-no-vendor-prefix': true,
     'selector-no-vendor-prefix': true,
     'value-no-vendor-prefix': true,
-    'string-quotes': 'single',
```

Comment:
This has been deprecated in Stylelint v15. We only needed to set this to make sure Stylelint didn't conflict with Prettier, but now Stylelint doesn't include it we don't need to. This will now _only_ be based on the prettierrc.


https://stylelint.io/user-guide/rules#deprecated